### PR TITLE
feat: update icon PR  details row

### LINKED ISSUE #696: Feature: Add PR details row data to generic profile page
### Type of feature

🍕 Feature

### Currently 
image

The PR details row was temporarily hidden due to shortage of data from the API, but the data needed for that row has been updated in the api and can be consumed from the frontend.

### Suggested solution

Unhide the row and implement the data features as seen in the image below
image

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR: 

- Updates the contributors profile page to include PRs related data
- Updates global `DbContribution` types with extra values from the api

### Notes:
The Open PRs percentage is still static and will be updated in a follow-up PR as it depend's on updates from the API
cc: @brandonroberts 

## Related Tickets & Documents
Fixes #696

## Mobile & Desktop Screenshots/Recordings

image

### Mobile
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
@SunGoldTech, looking at the OP mock and I think we should remove icons and percents altogether. They are not adding value. 

Please update the exact copy. 
- PRs Opened
- PR Velocity
- Contributed Repos 

PRs reviewed are not possible right now. 

Screen Shot 2022-12-22 at 5 30 28 AM

**Comment by SunGoldTech** [comment]
> PRs reviewed are not possible right now.

The data received from the API call for the `useSingleContributor` hook reflects a `recent_pr_reviews` field in the return object...

### Question:

- Is the `recent_pr_reviews` field not meant for the Prs Reviewed path?

cc: @bdougie

**Comment by brandonroberts** [comment]
The `recent_pr_reviews` is assigned reviews, not completed reviews. We can decide whether that's useful to show or not

File: components/molecules/CardRepoList/card-repo-list.tsx [hunk 1]
```diff
@@ -22,9 +22,9 @@ const CardRepoList = ({ repoList, limit = 5, fontSizeClassName }: CardRepoListPr
           {sanitizedRepoList
             .filter((repo, arrCount) => arrCount < limit)
             .map(({ repoName, repoIcon }, index) => (
-              <>
+              <div key={`${index * Math.random()}`}>
```
Comment (0-vortex) [inline components/molecules/CardRepoList/card-repo-list.tsx:25]: don't assign random keys please

File: components/organisms/ContributorProfilePage/contributor-profile-page.tsx [hunk 2]
```diff
@@ -83,47 +88,72 @@ const ContributorProfilePage = ({
               Contribution Insights
             </Title>
           </div>
-          <div className="bg-white mt-4 rounded-2xl border p-2 md:p-6">
-            <div className=" flex-col hidden lg:flex-row gap-2 md:gap-12 lg:gap-16 justify-between">
-              <div>
+          <div className="bg-white mt-4 rounded-2xl border p-4 md:p-6">
+            <div className=" flex flex-col lg:flex-row gap-2 md:gap-12 lg:gap-16 justify-between">
+              <div className="">
```
Comment (0-vortex) [inline components/organisms/ContributorProfilePage/contributor-profile-page.tsx:93]: this looks like a typo 🍕

File: pages/user/[username].tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ const Contributor = () => {
   const contributorLogin = username as string;
 
   const { data: contributor, isLoading: contributorLoading, isError } = useSingleContributor(contributorLogin);
-
+  console.log(contributor);
```
Comment (0-vortex) [inline pages/user/[username].tsx:18]: remove this as well please

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Some minor changes needed, but LGTM! 👍

**Comment by brandonroberts** [comment]
@OgDev-01 this one needs merge conflicts to be resolved also

**Review by 0-vortex — APPROVED** [review]

**Pull request merged**